### PR TITLE
Migrate intranet guidance

### PR DESCRIPTION
**Pull request by Jamie-Gibson**
In this PR I’ve added a number of guidelines that have been previously stored in the intranet, so they’re all together in one place with the other guidelines. 

There are a number of other guidelines related to coding that haven’t yet been added because I don’t know the status of them. Assigning to David G, Miguel, Inga and Simao to help with finalising those guidelines when they have time

https://sites.google.com/a/vizzuality.com/intranet/guidelines/best-practices
https://sites.google.com/a/vizzuality.com/intranet/guidelines/code-review
https://sites.google.com/a/vizzuality.com/intranet/guidelines/code-style
https://sites.google.com/a/vizzuality.com/intranet/guidelines/front-end-workflow

### CODE REVIEW

File: guidelines/claiming-expenses.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#**Claiming Expenses**
```
Comment (simaob) [inline guidelines/claiming-expenses.md]: Should we update this to include Porto?

**Comment by simaob** [comment]
Other than that comment, it looks good to me. =)

File: guidelines/claiming-expenses.md (hunk 1 above)
Comment (agnessa) [reply guidelines/claiming-expenses.md]: I updated the text to incorporate Porto - does that look good @d4weed?

**Comment by d4weed** [comment]
the google analytics guideline seems a bit orphan. It looks like it could be included in the project workflow somehow if it still isn't

**Comment by d4weed** [comment]
shall we merge this?

**Comment by simaob** [comment]
I think so, yes

**Pull request merged**